lib/ccf/functional_preprocessing/one_subject_run_status_checker.py

```diff
@@ -7,6 +7,9 @@
 # import of local modules
```

Comment:
I think all the changes in this file are for the other feature: invoking functions from a local system instead of from the CHPC